docs/Tutorials/Multifile.md

```diff
@@ -6,6 +6,8 @@ description: Signing transactions with multiple keys.
 
 EthSigner supports file-based signing using [multiple keys](../Concepts/Multiple-Key-Files.md).
 
+This tutorial covers configuring multiple keys using a V3 keystore file. To configure multiple keys for Hashicorp Vault or Azure Key Vault, update the [TOML configuration file](../Reference/Multikey-Parameters.md) accordingly.
```

Comment:
Suggest adding links to the Store Key Externally -> Use Hashicorp Vault and Use Azure Key Vault topics on Hashicorp Vault and Azure Key Vault

Reply:
Done